packages/shared/src/components/RenderMarkdown.tsx

```diff
@@ -137,23 +141,46 @@ const RenderMarkdown = ({
             loadLanguages();
           }
 
-          return !inline && language ? (
-            <SyntaxHighlighterAsync
-              {...props}
-              customStyle={containerReset}
-              language={language}
-              useInlineStyles={false}
-              wrapLongLines
-              codeTagProps={{
-                className: codeClassName,
-              }}
-            >
-              {String(children).replace(replaceNewLineRegex, '')}
-            </SyntaxHighlighterAsync>
-          ) : (
-            <code {...props} className={codeClassName}>
-              {children}
-            </code>
+          return (
+            <>
+              <div className="flex justify-between py-2 px-5 bg-theme-active align-center">
+                <span className="inline leading-8 text-theme-label-tertiary">
+                  {language}
+                </span>
+                <Button
+                  className="btn-tertiary"
+                  icon={<CopyIcon />}
+                  disabled={copying}
+                  buttonSize={ButtonSize.Small}
```

Comment:
Was small ok?
Think the design states `XSmall` it should be a 20x20 button size.

Reply:
Felt too small, but you're right, updated as per spec